web/email.ex

```diff
@@ -18,6 +18,22 @@ defmodule Community.Email do
     |> render("job_posted.text", job: job)
   end
 
+  def admin_member_added(member) do
+    new_email
+    |> to(System.get_env("ADMIN_EMAILS"))
```

Comment:
What does `ADMIN_EMAILS` look like? If it's something like `me@example.com, someone_else@example.com` then you'll have to do `String.split` on it first